draft-ietf-quic-transport.md

```diff
@@ -4636,10 +4636,8 @@ Version Negotiation Packet {
 ~~~
 {: #version-negotiation-format title="Version Negotiation Packet"}
 
-The value in the Unused field is selected randomly by the server.  Clients MUST
-ignore the value of this field.  Servers SHOULD set the most significant bit of
-this field (0x40) to 1 so that Version Negotiation packets appear to have the
-Fixed Bit field.
+The value in the Unused field is set to an arbitrary value by the server.
```

Comment:
I guess it would make sense to note that this bit can not be assumptned to be 1 for VN packets also earlier in the document as the section on long header packets says: 

```
The next bit (0x40) of byte 0 is set to 1. Packets containing a zero value for this bit are not valid packets in this version and MUST be discarded.
```